styx-scheduler-service/src/main/java/com/spotify/styx/BackfillTriggerManager.java

```diff
@@ -113,7 +111,14 @@ void tick() {
   }
 
   private void triggerAndProgress(Backfill backfill) {
-    final Optional<Workflow> workflowOpt = workflowCache.workflow(backfill.workflowId());
+    final Optional<Workflow> workflowOpt;
+    try {
+      workflowOpt = storage.workflow(backfill.workflowId());
+    } catch (IOException e) {
+      LOG.warn("Failed to read workflow {}", backfill.workflowId(), e);
```

Comment:
Before this change, the workflow cache would throw a RTE that would bubble up to the caller of `triggerAndProgress` but now the caller will not get an exception. Does that matter?